test/test_slmigrate.py

```diff
@@ -4,6 +4,7 @@
 from nislmigrate.argument_handler import RESTORE_ARGUMENT
 from nislmigrate.argument_handler import MIGRATION_DIRECTORY_ARGUMENT
 from nislmigrate.facades.mongo_facade import MongoFacade
+from nislmigrate.logs.migration_error import MigrationError
```

Comment:
Do you need this?

Reply:
I do now that I added a test at this level.